kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -66,8 +66,21 @@
   private static final ConfigDef.Width TOPICS_WIDTH =            ConfigDef.Width.LONG;
   private static final String TOPICS_DOC =
       "List of topics to consume, separated by commas";
+  public static final String TOPICS_DEFAULT = "";
   private static final String TOPICS_DISPLAY =                   "Topics";
 
+  public static final String TOPICS_REGEX_CONFIG =                     "topics.regex";
+  private static final ConfigDef.Type TOPICS_REGEX_TYPE =              ConfigDef.Type.STRING;
```

Comment:
When many different topics configured for one KCBQ connector, it will be odd to represent them all with single regex. How about make it a list of regex?

Reply:
Aha, looks like these config are mirroring https://github.com/apache/kafka/blob/1.1.1/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/SinkConnectorConfig.java#L38-L43